server/base/src/main/java/org/apache/accumulo/server/manager/LiveTServerSet.java

```diff
@@ -314,24 +321,17 @@ public void process(WatchedEvent event) {
       if (event.getPath().endsWith(Constants.ZTSERVERS)) {
         scanServers();
       } else if (event.getPath().contains(Constants.ZTSERVERS)) {
```

Comment:
Would this work here?  Was trying to figure out how to leverage the new code here.

```java
  } else {
      try{
          final ServiceLockPath slp = ServiceLockPaths.parse(Optional.empty(), event.getPath());
          if (slp.getType().equals(Constants.ZTSERVERS)) {
```

Reply:
Yeah, good thought. I did this in 48e8662.